test/livebook_web/live/hub/edit_live_test.exs

```diff
@@ -81,9 +81,8 @@ defmodule LivebookWeb.Hub.EditLiveTest do
       |> render_submit(attrs)
 
       assert_receive {:secret_created, ^secret}
-
-      %{"success" => "Secret PERSONAL_ADD_SECRET added successfully"} =
-        assert_redirect(view, "/hub/#{hub.id}")
+      assert_patch(view, "/hub/#{hub.id}")
+      assert render(view) =~ "Secret PERSONAL_ADD_SECRET added successfully"
 
       {:ok, view, _html} = live(conn, ~p"/hub/#{hub.id}")
```

Comment:
We already assert the patch, so we should make content assertions against the updated page, rather than reloading (this should reproduce the delete secret issue on test level):

```suggestion
```

(same in all other places)